netcore/Makefile

```diff
@@ -234,6 +239,36 @@ build-test-corefx-%:
 xunit-summary:
 	./xunit-summary.py "corefx/tests"
 
+
+update-tests-coreclr: coreclr/.stamp-tests
+
+coreclr/.stamp-tests:
+	rm -rf coreclr/tests
+	curl -L "$(CORECLR_TESTS)" --output coreclr/tests.zip
+	$(UNZIPCMD) coreclr/tests.zip -C coreclr/
```

Comment:
It downloads a `.zip` file but tries to extract it with `tar` on Linux which fails.